content/blog/newsletter/2025-12-05-newsletter.md

```diff
@@ -0,0 +1,195 @@
+
+{{< figure src="/images/newsletter/2025-12-05/movim_post.png" caption="XMPP and ActivityPub: Two Different Approaches." >}}
+
+- [Introducing Borogove, the new Snikket SDK](https://snikket.org//blog/borogove-the-new-snikket-sdk/) by the Snikket Team for the [Snikket blog](https://snikket.org/blog/).
+- [Best Python XMPP/Jabber Client Libraries: Your Experiences & Top Recommendations](https://www.pythontutorials.net/blog/best-python-xmpp-jabber-client-library/) from [PythonTutorials.net Blog](https://www.pythontutorials.net/blog/).
+
+## XMPP Software News
+
+### XMPP Clients and Applications
+
+- [Dino](https://dino.im/) has released version [0.5.1](https://github.com/dino/dino/releases/tag/v0.5.1) of it's a modern open-source chat client for the desktop. It focuses on providing a clean and reliable Jabber/XMPP experience while having your privacy in mind.
+- [Gajim](https://gajim.org/) has released version [2.4.0](https://gajim.org/posts/2025-11-04-gajim-2.4.0-released/) of its free and fully featured chat app for XMPP. This release brings read markers in group chats, improved file transfers, more details for your account, and many smaller changes and bug fixes. Thank you for all your contributions! You can take a look at the [changelog](https://dev.gajim.org/gajim/gajim/-/blob/master/ChangeLog) for all the details.
+
+{{< figure src="/images/newsletter/2025-12-05/gajim-profile.png" caption="Gajim: Your profile on the account page." >}}
+
+- [Monal](https://monal-im.org/) has released version [6.4.14](https://github.com/monal-im/Monal/releases/tag/Build_iOS_1053) and  for iOS and macOS.
+- [Monocles](https://monocles.eu/) has released version [2.0.17](https://codeberg.org/monocles/monocles_chat/releases/tag/CodebergRelease-v2.0.17) of its chat client for Android. This update brings some fixes like more visible confirmation icon, update and add photo filter library, fix photo editor menu overlapping with status bar, setting to always use relays, and an improved extension deletion logic among many others. Make sure to take a look at the [changelog](https://codeberg.org/monocles/monocles_chat/src/branch/master/fastlane/metadata/android/en-US/changelogs/19401.txt) for all the details!
+- [Movim](https://movim.eu/) has released version [0.32](https://mov.im/community/pubsub.movim.eu/Movim/here-comes-movim-0-32-codename-wilk-CBbSYh), code named '[Wilk](https://en.wikipedia.org/wiki/C/1925_V1_(Wilk%E2%80%93Peltier))', with version [0.32.1](https://github.com/movim/movim/releases/tag/v0.32.1) immediately following it. This might be one of the biggest releases ever made on the project. It includes numerous fixes and new features in the 3 major topics [Movim](https://movim.eu/) offers: instant messaging, social networking, and video conferencing. Head over the [official release announcement](https://mov.im/community/pubsub.movim.eu/Movim/here-comes-movim-0-32-codename-wilk-CBbSYh) and dive directly into the exciting things you can find in this release!
```

Comment:
```suggestion
- [Movim](https://movim.eu/) has released version [0.32](https://mov.im/community/pubsub.movim.eu/Movim/here-comes-movim-0-32-codename-wilk-CBbSYh), code named '[Wilk](https://en.wikipedia.org/wiki/C/1925_V1_(Wilk%E2%80%93Peltier))', with version [0.32.1](https://github.com/movim/movim/releases/tag/v0.32.1) immediately following it. This might be one of the biggest releases ever made on the project. It includes numerous fixes and new features in the 3 major topics [Movim](https://movim.eu/) offers: instant messaging, social networking, and video conferencing. Head over to the [official release announcement](https://mov.im/community/pubsub.movim.eu/Movim/here-comes-movim-0-32-codename-wilk-CBbSYh) and dive directly into the exciting things you can find in this release!
```